WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -32,8 +45,131 @@ struct WooShippingCreateLabelsView: View {
 
                     WooShippingPackageAndRatePlaceholder()
                 }
+                .padding()
+            }
+            .safeAreaInset(edge: .bottom) {
+                ExpandableBottomSheet(onChangeOfExpansion: { isExpanded in
+                    isShipmentDetailsExpanded = isExpanded
+                }) {
+                    if isShipmentDetailsExpanded {
+                        VStack(spacing: Layout.bottomSheetSpacing) {
+                            Toggle(Localization.BottomSheet.markComplete, isOn: .constant(false)) // TODO: 14044 - Handle this toggle setting
+                                .font(.subheadline)
+
+                            Menu {
+                                ForEach(paperSizes, id: \.self) { paperSize in
+                                    Button {
+                                        selectedPaperSize = paperSize
+                                    } label: {
+                                        HStack {
+                                            Text(paperSize)
+                                            if selectedPaperSize == paperSize {
+                                                Image(uiImage: .checkmarkStyledImage)
+                                            }
+                                        }
+                                    }
+                                }
+                            } label: {
+                                HStack {
+                                    Text(selectedPaperSize)
+                                        .bodyStyle()
+                                        .frame(maxWidth: .infinity, alignment: .leading)
+                                    Image(systemName: "chevron.up.chevron.down")
+                                        .bold()
+                                }
+                                .padding(insets: EdgeInsets(top: 12, leading: 16, bottom: 12, trailing: 16))
+                            }
+                            .roundedBorder(cornerRadius: Layout.cornerRadius, lineColor: Color(.separator), lineWidth: 1)
+                            .accessibilityLabel(Localization.BottomSheet.paperSize)
+
+                            Button {
+                                // TODO: 13556 - Trigger purchase action
+                            } label: {
+                                Text(Localization.BottomSheet.purchase)
+                            }
+                            .buttonStyle(PrimaryButtonStyle())
+                            .disabled(true) // TODO: 14044 - Enable button when shipping label is ready to purchase
```

Comment:
We might want to handle this when we integrate the purchase endpoint (#13556), so we can account for/validate all the of the data required to make the purchase.